src/interface/src/app/services/auth.service.ts

```diff
@@ -133,19 +133,45 @@ export class AuthService {
       );
   }
 
-  changePassword(password1: string, password2: string): Observable<any> {
+  changePassword(currentPassword: string, newPassword1: string, newPassword2: string): Observable<any> {
     return this.http.post(
       this.API_ROOT.concat('password/change/'),
       {
-        new_password1: password1,
-        new_password2: password2,
+        old_password: currentPassword,
+        new_password1: newPassword1,
+        new_password2: newPassword2,
       },
       {
         withCredentials: true,
       }
     );
   }
 
+  sendPasswordResetEmail(userId: string, email: string): Observable<any> {
+    return this.http.post(
+      this.API_ROOT.concat('password/reset/'),
```

Comment:
The POST request needs to be sent to `/planscape-backend/dj-rest-auth/password/reset/`

Reply:
This is where it is currently being sent correct.